ci/helm/helmfile.yaml

```diff
@@ -15,27 +17,44 @@ releases:
     chart: nuxeo/nuxeo
     version: 3.0.11
     needs:
-      - elasticsearch
-      - mongodb
+      - {{ eq .Environment.Name "functional-tests-2023" | ternary "opensearch" "elasticsearch" }}
+      - {{ eq .Environment.Name "functional-tests-2023" | ternary "mongodb6" "mongodb4" }}
```

Comment:
It could be better to have the condition on the `elasticsearch.enabled` and `opensearch.enabled`, same for mongodb.

Reply:
Why is it resolved? The condition is still on the env name 🤔

Reply:
It has been rollbacked because https://github.com/roboll/helmfile/issues/1501#issuecomment-699528468